pkg/api/roles.go

```diff
@@ -198,17 +198,35 @@ func (hs *HTTPServer) declareFixedRoles() error {
 			Group:       "Teams",
 			Version:     1,
 			Permissions: []accesscontrol.Permission{
-				{
-					Action: accesscontrol.ActionTeamsCreate,
-				},
+				{Action: accesscontrol.ActionTeamsCreate},
 			},
 		},
 		Grants: teamCreatorGrants,
 	}
 
+	teamsWriterRole := accesscontrol.RoleRegistration{
+		Role: accesscontrol.RoleDTO{
+			Name:        "fixed:teams:writer",
+			DisplayName: "Team writer",
+			Description: "Create, read, write, or delete a team as well as controlling team memberships.",
+			Group:       "Teams",
+			Version:     1,
+			Permissions: []accesscontrol.Permission{
+				{Action: accesscontrol.ActionTeamsCreate},
```

Comment:
question: should we include team creation in this role or not? It's already granted as part of teams creator role, and it might be confusing to have it as part of two fixed roles. No strong opinions on this though.